nextflow.config

```diff
@@ -1,8 +1,78 @@
-singularity.enabled = false
-singularity.autoMounts = false
+profiles {
 
-docker.enabled = true
-process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  conda {
+    executor = 'local'
+    singularity.enabled = false
+    docker.enabled = false
+    conda.enabled = true
+  }
+
+  docker {
+    executor = 'local'
+    singularity.enabled = false
+    singularity.autoMounts = false
+
+    docker.enabled = true
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+  }
+
+  sanger_lsf {
+    singularity.enabled = true
+    singularity.autoMounts = false
+    process.container = 'haniffalab/webatlas-pipeline:0.3.2'
+
+    docker.enabled = false
+    singularity {
+        enabled = true
+        autoMounts = false
+        runOptions = '--bind /lustre --bind /nfs --cleanenv --home /tmp'
+    }
+    executor {
+        name = 'lsf'
+        perJobMemLimit = true
+        container = '/lustre/scratch127/cellgen/cellgeni/tickets/tic-2248/actions/docker--haniffalab--vitessce-pipeline--0.0.1.sif'
+        jobName = { "$task.hash" }
+        queue = 'imaging'
+    }
+  }
+}
+
+process {
+
+    withName: image_to_zarr {
+	conda = '/opt/conda/envs/webatlas-0.3.2'
```

Comment:
if you add to the **conda** profile instead of to each processs here like:
```
conda {
    process.conda = '/some/path/environment.yaml'
  }
```

does it get re-created when each process start or nextflow is smarter than that?

Reply:
I need to give it a try. I am pretty sure it is caching the env. But I need to check two things first:
1. if we can give the `yaml` file directly here
2. For the first run, when there's no cached envs. Will it create several `same new env`.
Thanks for looking at all above!

Reply:
Using projectDir/docker/conda.yaml for relative import. Done